src/lti/LTI_Constants.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace IMSGlobal\LTI;
+
+class LTI_Constants
```

Comment:
This is super neat and something i've been meaning to do for an age but never brought myself to do it, so thanks for that